src/main/java/com/syncstream_dev/syncstream/model/message/UserMessage.java

```diff
@@ -0,0 +1,22 @@
+package com.syncstream_dev.syncstream.model.message;
+
+import com.syncstream_dev.syncstream.model.user.User;
+
+public class UserMessage extends Message {
+    private User sender;
+    private Boolean flag;
+
+    public UserMessage(String mid, long timestamp, String content, User sender, Boolean flag) {
+        super(mid, timestamp, content);
+        this.sender = sender;
+        this.flag = flag;
+    }
+
+    public User getSender() {
+        return sender;
+    }
+
+    public Boolean flag() {
```

Comment:
Functions that return a boolean type should be prefixed with `is` or `has`.

Reply:
7557948eb4b887d6619f0647c8e9d56e817770de addresses this